typescript/configuration.md

```diff
@@ -21,6 +21,12 @@ a path on the command line. For example:
 > other TypeScript modules incompatible. Honestly it is best to use the Deno
 > defaults and to think long and hard about using a configuration file.
 
+> ⚠️ With v1.14 Deno started supporting more general configuration file, that is
```

Comment:
Typo on "specyfing" and maybe move the comma over: "With v1.14, Deno started supporting a more general configuration file that is no longer confined to specifying TypeScript compiler settings."